crates/hc_sandbox/src/zome_call.rs

```diff
@@ -0,0 +1,516 @@
+//! ## Making zome calls with the `hc sandbox zome-call` command
+//!
+//! To get started, you need a running conductor with an app installed. This example uses a sandbox
+//! conductor with a single app.
+//!
+//! ```shell
+//! hc sandbox generate --run=0 ./my-app.happ --app-id my-app
+//! ```
+//!
+//! Enter a passphrase for the conductor when prompted. This passphrase is used to protect the Lair
+//! keystore and an encryption key for Holochain data is derived from it.
+//!
+//! Now you have a running conductor, switch to a new shell and `cd` to the same directory that
+//! you started the sandbox in. You should find a `.hc` file in this directory.
+//!
+//! Next we need to authorize zome calls for the app. Do this by running the following command,
+//! providing the installed app id of the app you want to authorize zome calls for.
+//!
+//! ```shell
+//! hc sandbox zome-call-auth my-app
```

Comment:
If I run this command without explicitly providing the admin port with the `-r` option but in the directory with the `.hc_live_1` file containing the correct admin port I get the error:
```
$ hc-sandbox zome-call-auth my-app
Error: IO error: Connection refused (os error 111)

Caused by:
    Connection refused (os error 111)
```

Reply:
never mind, this may also have been due to an old conductor in the .hc file. It's working now after `hc-sandbox clean`.